test/Azure.Functions.Cli.Tests/Azure.Functions.Cli.Tests.csproj

```diff
@@ -49,6 +49,10 @@
     <ProjectReference Include="..\..\src\Azure.Functions.Cli\Azure.Functions.Cli.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="E2E\TestProject\" />
```

Comment:
This can be deleted. VS adds these `Folder` items when you delete the last file in a folder. The only thing is accomplishes is showing the empty folder in solution explorer. Fun legacy behavior from pre-SDK csproj 😄